package.json

```diff
@@ -35,9 +37,9 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@commitlint/cli": "^11.0.0",
-    "@commitlint/config-conventional": "^11.0.0",
-    "@commitlint/config-lerna-scopes": "^11.0.0",
+    "@commitlint/cli": "^17.0.0",
```

Comment:
This bump was needed because older version had a peer dependency on a lower version of Lerna. There don't seem to be any breaking changes.